sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -50,15 +49,15 @@ internal class DigitalTwinsLifecycleSamples
         private static readonly string s_twinsPath = Path.Combine(s_dtdlDirectoryPath, "DigitalTwins");
         private static readonly string s_relationshipsPath = Path.Combine(s_dtdlDirectoryPath, "Relationships");
 
-        private readonly string _eventhubEndpointName;
         private readonly string _eventRouteId = $"sampleEventRouteId-{Guid.NewGuid()}";
 
-        private DigitalTwinsClient DigitalTwinsClient { get; }
+        private readonly string eventhubEndpointName;
+        private readonly DigitalTwinsClient client;
```

Comment:
This is a compromise for now. It can't just be a local variable, unless we pass it from each method to the next. It breaks the naming convention, but is less work.